components/projects/form/fields/GenerationField.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from '@emotion/styled';
+import React, { FC, useState } from 'react';
+
+import Checkbox from '@/components/common/Checkbox';
+import ErrorMessage from '@/components/common/Input/ErrorMessage';
+import Select from '@/components/common/Select';
+import Text from '@/components/common/Text';
+import { GENERATIONS } from '@/constants/generation';
+import { colors } from '@/styles/colors';
+import { MOBILE_MEDIA_QUERY } from '@/styles/mediaQuery';
+import { textStyles } from '@/styles/typography';
+
+interface GenerationFieldProps {
+  value: string;
+  onChange: (value: string) => void;
+  isError?: boolean;
+  errorMessage?: string;
+}
+
+const GenerationField: FC<GenerationFieldProps> = ({ value, onChange, isError, errorMessage }) => {
+  const [checked, setChecked] = useState<boolean>(!!value);
+
+  const handleGenerationChange = (generation: string) => {
+    if (generation) {
+      setChecked(false);
+    }
+    onChange(generation);
+  };
+
+  const handleCheckedChange = (checked: boolean) => {
+    if (checked) {
+      onChange('');
+    }
+    setChecked(checked);
+  };
+
+  return (
+    <StyledGenerationField>
+      <StyledDescription>참여한 팀원들의 기수에 맞춰 작성해주세요</StyledDescription>
+      <StyledSelect
+        width={236}
+        placeholder='선택'
+        value={value}
+        onChange={(e) => handleGenerationChange(e.target.value)}
+      >
+        {GENERATIONS.map((item) => (
+          <option key={item} value={item}>
+            {item}기
+          </option>
+        ))}
+      </StyledSelect>
+      <StyledCheckboxWrapper>
+        <Checkbox checked={checked} onChange={(e) => handleCheckedChange(e.target.checked)} />
+        <Text typography='SUIT_12_M' color={colors.gray100}>
+          특정 기수 활동으로 진행하지 않았어요
+        </Text>
+      </StyledCheckboxWrapper>
+      {isError && <StyledErrorMessage message={errorMessage} />}
```

Comment:
p3; ErrorMessage 컴포넌트에는 메시지가 비어있을 때 에러를 표시하지 않고, 공간만 차지하게 하는 기능이 내장되어 있어요.
isError로 구별하지 않고 따로 넣으면 더 밑쪽의 필드들이 에러가 표시될 때 일렁이는 현상을 방지할 수 있어요!

```suggestion
      <StyledErrorMessage message={errorMessage} />
```

Reply:
[4d58722](https://github.com/sopt-makers/sopt-playground-frontend/pull/471/commits/4d587228f3842d1c545d6352b651fec1559ae559) 해당 커밋에서 같이 처리했습니다!